src/lib/protocols/http.c

```diff
@@ -589,8 +589,10 @@ static u_int16_t http_request_url_offset(struct ndpi_detection_module_struct *nd
   int i;
 
   NDPI_LOG_DBG2(ndpi_struct, "====>>>> HTTP: %c%c%c%c [len: %u]\n",
-		non_ctrl(packet->payload[0]), non_ctrl(packet->payload[1]),
```

Comment:
Why removing `non_ctrl` ?

Reply:
I didn't. The function non_ctrl() has been removed on 8b1c68ae, one year ago!
This is the perfect example to show how the debug message stuff has been broken for a long time. If --enable-debug-messages flag had worked as intended, anyone compiling ndpi with such flag in the last year would have received a compiler error.
I don't think that for such low-level message we need non_ctrl() again, but if anyone thinks otherwise, I could definitely reintroduce it